presto-parser/src/main/antlr4/com/facebook/presto/sql/parser/SqlBase.g4

```diff
@@ -705,6 +719,14 @@ DOUBLE_PRECISION
     : 'DOUBLE' WS 'PRECISION'
     ;
 
+fragment ESCAPE_INTRODUCER
```

Comment:
No need for a separate rule for this. It's used in a single place. Take a look at how BINARY_LITERAL works.